chromadb/api/types.py

```diff
@@ -71,6 +71,17 @@ def maybe_cast_one_to_many(target: Optional[OneOrMany[T]]) -> Optional[List[T]]:
 Space = Literal["cosine", "l2", "ip"]
 
 
+def embeddings_to_base64_bytes(pyEmbeddings: PyEmbeddings) -> tuple[str, list[int]]:
+    array = np.array(pyEmbeddings, dtype=np.float32)
```

Comment:
You don't need to use np here - you can use https://docs.python.org/3/library/struct.html and get a little endian repr and put that in